Mtneer/mushu
# View all transactions

# Description
> As an user, I would like to see a list of all the Transactions so that I can monitor my expenses.
> 
> **Given** the user is in the Mushu application
> **When** they select the `Transactions` menu option
> **Then** they should be directed to the Transactions list page
> **And** each transaction in the list should display the date, retailer, type, category, and amount
> **And** the list should be in order of transaction date with the most recent on top

## Type of change
Please delete options that are not relevant.

# Testing Instructions
- follow all directions on prior PR to set up environment if you have not yet done so
- pull down new branch with "git fetch -a" and "git checkout view-all-transactions"
- start your react app using "npm start" from the client directory
- start your web api by clicking play from Visual Studio
- login to the app using "admin@admin.com" "password1"
- Click on the "Transactions" tab in the header menu
- Verify that a list of transactions appears in order of descending transaction date

# Checklist:
